### ISSUE TITLE
Cannot read property 'showModal' of undefined

### ISSUE THREAD

**Issue by pautiina** [clonos/cp#12]
При попытке открыть хэлперы или экспортировать виртуальную машину в консоле браузера вылазит следующая ошибка:
```
Uncaught TypeError: Cannot read property 'showModal' of undefined
    at Object.dialogShow1 (clonos.js:235)
    at Object.DDMenuSelect (clonos.js:2144)
    at Object.bodyClick (clonos.js:1635)
    at HTMLDivElement.e (jquery.js:2)
    at HTMLDivElement.dispatch (jquery.js:3)
    at HTMLDivElement.q.handle (jquery.js:3)
```
и естественно функция не открывается

**Comment by moveee** [comment]
Какой браузер используете? Открытие окон проверялось почти во всех. Данную возможность, насколько помню, только FireFox не поддерживал.

**Comment by olevole** [comment]
@moveee у меня в FF вроде как норм было год назад, а больше не проверялось. Надо восстановить разработку и стенд), мне лично пока негде проверить

**Comment by moveee** [comment]
В FF имитация была сделана. Но я посмотрю, может за год они уже допилили эту фичу в браузере. Вроде что-то есть.

**Comment by pautiina** [comment]
Проверялось на Опере.

**Comment by moveee** [comment]
А, ну понятно. В Опере не проверял. По-моему в ней не всё поддерживается, поэтому я в ней даже не тестирую. Потом установлю и попробую сделать.

**Comment by pautiina** [comment]
Да, только опера на движке хрома. В понедельник на остальных проверю

**Comment by pautiina** [comment]
Проверил сегодня в моззиле-та же ошибка

**Comment by moveee** [comment]
Только что проверил в FireFox: 62.0.3 (64-бит)
Открывается без проблем. В Хроме тоже.
Такое ощущение, что опубликована какая-то промежуточная версия интерфейса. Сейчас Оперу поставлю, посмотрю что в ней.

**Comment by moveee** [comment]
Кстати, у меня Windows 10, если что. Если не работает на другой ОС, то это нужно Олегу тогда проверить.

**Comment by moveee** [comment]
Я извиняюсь. Похоже неправильно понял с самого начала. Думал вообще диалоговые окна не открываются.
Экспорт машины ещё не реализован. Там нет диалогового окна. Хелперы, похоже, тоже. Это мы ещё не успели сделать.

**Comment by pautiina** [comment]
Тогда всё понятно. Будем ждать нового функционала
